features/FEATURE_LWIP/lwip-interface/ppp_lwip.cpp

```diff
@@ -71,7 +71,11 @@ static EventQueue *prepare_event_queue()
 
     // Only need to queue 1 event. new blows on failure.
     event_queue = new EventQueue(2 * EVENTS_EVENT_SIZE, NULL);
-    event_thread = new Thread(osPriorityNormal, 900);
+#if LWIP_DEBUG
+    event_thread = new Thread(osPriorityNormal, 900*2);
```

Comment:
Trace has always required double stack in lwIP - the debug changes here are actually reducing the PPP trace stack, which was even worse than that.